code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -149,7 +156,6 @@
 
 //We're done working on mobs, let's check if we're blessing something else
 /obj/item/weapon/storage/bible/afterattack(atom/A, mob/user as mob)
-	user.delayNextAttack(5) //Extra delay
```

Comment:
Third time I've said this, why is this being removed?
If your answer is "afterattack doesn't work" then that's not a good reason, re-add it